registry/client/src/lib.rs

```diff
@@ -30,9 +30,12 @@ lazy_static::lazy_static! {
         vault_signer: Pubkey::new_from_array([0; 32]).into(),
         vault_signer_nonce: 0,
         account_params: vec![],
+        serum_fee_vault: Pubkey::new_from_array([0; 32]).into(),
```

Comment:
The latest updates on master removes the registry's dependency on the pool crate FYI.